### PR TITLE
don't set bootstrap toggle on ToggleButtonsWidget

### LINKED ISSUE #6562: Toggle buttons' state not displayed correctly
```Python
from IPython.html.widgets import ToggleButtons
from collections import OrderedDict
from IPython.display import display
ToggleButtons(values=OrderedDict([('a', 1), ('b', 2), ('c', 3)]))
```
From the open-source days, the  toggle buttons don't always reflect the selected one. 
Sylvain

### PR DESCRIPTION
**Pull request by minrk**
Model update already toggles active and data-toggle does the same, so clicking on a toggle toggles twice, setting it back to the original value.

closes #6562

### CODE REVIEW

**Comment by Carreau** [comment]
faillure seem real.

**Comment by minrk** [comment]
Waiting for Travis, but should be fixed.

File: IPython/html/tests/widgets/widget_selection.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ casper.notebook_test(function () {
     this.execute_cell_then(index);
 
     var combo_selector = '.widget-area .widget-subarea .widget-hbox .btn-group .widget-combo-btn';
-    var multibtn_selector = '.widget-area .widget-subarea .widget-hbox .btn-group[data-toggle="buttons-radio"]';
+    var multibtn_selector = '.widget-area .widget-subarea .widget-hbox.widget-toggle-buttons .btn-group';
```
Comment (takluyver) [inline IPython/html/tests/widgets/widget_selection.js]: Should there be a space in `.widget-hbox.widget-toggle-buttons`?

Comment (minrk) [inline IPython/html/tests/widgets/widget_selection.js]: No, space indicates heirarchy. No space indicates one element with two classes.

(commas!)

**Comment by Carreau** [comment]
Let's merge it to get complains if any.

**Pull request merged**